controllers/mustgather/mustgather_controller_test.go

```diff
@@ -77,7 +77,7 @@ func createMustGatherObject() *mustgatherv1alpha1.MustGather {
 				Name: "case-management-creds",
 			},
 			ServiceAccountRef: corev1.LocalObjectReference{
-				Name: "must-gather-admin",
+				Name: "",
```

Comment:
This test used to rely on "MustGatherTimeout" not being initialized yet to check if the Reconcile loop initialized the object. Now that it will always have a valid default value, I had to force something else to be uninitialized.